coretex/entities/sample/network_sample.py

```diff
@@ -112,6 +114,31 @@ def _createSample(
 
         return cls.decode(response.json)
 
+    def modifiedSinceLastDownload(self) -> bool:
+        """
+            Checking if sample has been modified since last download, if the sample is already
+            stored locally
+
+            Returns
+            -------
+            bool -> False if sample has not changed since last download, True otherwise
+        """
+
+        if not self.zipPath.exists():
+            return False
+
+        sampleJson = self.fetchById(self.id).encode()
+        lastModified = sampleJson["storage_last_modified"]
+
+        if lastModified is None:
+            return False
```

Comment:
Why is it possible for `storage_last_modified` to be `None`?

Reply:
This is a common value found in testing

Reply:
Has this been confirmed with the backend team?

Reply:
Backend has confirmed that None is the expected value if the object has never been modified

Reply:
Let's remove this None check and modify this to not be nullable